.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -45,9 +45,15 @@ jobs:
           - host_os: cbl-mariner
             vmm: clh
             instance-type: small
+            genpolicy-pull-method: oci-distribution
+          - host_os: cbl-mariner
+            vmm: clh
+            instance-type: small
+            genpolicy-pull-method: containerd
           - host_os: cbl-mariner
             vmm: clh
             instance-type: normal
+            genpolicy-pull-method: oci-distribution
```

Comment:
Do we not need the equivalent of this entry with the small instance type?

Reply:
Not needed since no test for "normal" instance-type autogenerate policy. Actually I just removed this line as well